packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
+    <ul class="hds-tab-list__list" role="tablist">
+      {{! Hds-Tab }}
+      <li class="hds-tab" role="presentation">
+        <a
+          role="tab"
+          href="#section1"
+          id="tab1"
+          {{! aria-selected="true"  TODO: Set by arg: "isSelected" }}
+          {{! tabindex="-1"  TODO: Set if not aria-selected ("isSelected") }}
+        >
+          Tab 1
+          {{this.isSelected}}
+        </a>
+      </li>
+      <li class="hds-tab" role="presentation">
+        <a
+          role="tab"
+          href="#section2"
+          id="tab2"
+          {{! aria-selected="true"  TODO: Set by arg: "isSelected" }}
+          {{! tabindex="-1"  TODO: Set if not aria-selected ("isSelected"), Focus will be added by JS }}
+        >
+          Tab 2
+          {{this.isSelected}}
+        </a>
+      </li>
+    </ul>
+  </div>
+
+  {{! Hds-TabPanel }}
+  <section
+    class="hds-tab-panel"
+    role="tabpanel"
+    id="section1"
+    aria-labelledby="tab1"
+    {{! hidden }}
+    {{! TODO: set by arg }}
+  >
+    Tab 1 Content
+  </section>
+  <section
+    class="hds-tab-panel"
+    role="tabpanel"
+    id="section2"
+    aria-labelledby="tab2"
+    {{! hidden }}
+    {{! TODO: set by arg }}
+  >
+    Tab 2 Content
+  </section>
+</div>
+
+{{!
+Component Structure:
+
+Tabs
+  TabList
+    Tab
+      Content:
+        FlightIcon (optional)
+        text content (required)
+        BadgeCount (optional)
+      Args:
+        isSelected (boolean, default = false) // Select first tab by default?
+        hasIcon (boolean, default = false)
+        hasCount (boolean, default = false)
+    Tab (2+ required)
+    ...
```

Comment:
I don't know yet if that needs to be a separate component. I'll break it out later when/if needed. I was trying to capture the main structure

Reply:
> I don't know yet if that needs to be a separate component
Unless I'm missing something, there's no other way: it's shared between tabs, and its `left/right` position values are set via JS (so it can't be a pseudo element; unless we use local CSS props, but in that case I don't know if the animation would work... 🤔)

Reply:
Ok fine but this was meant to be a preliminary break down of the main components and it was just a comment I included to think through things as I worked. I'm going to delete the comment eventually

Reply:
I updated my comment to clarify it's for temporary rough planning purposes only and not to meant to literally be the final implementation